Marlin/SanityCheck.h

```diff
@@ -1146,7 +1146,7 @@ static_assert(1 >= 0
 static_assert(1 >= 0
   #if ENABLED(ULTIMAKERCONTROLLER) \
       && DISABLED(SAV_3DGLCD) && DISABLED(miniVIKI) && DISABLED(VIKI2) \
-      && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER) && DISABLED(PANEL_ONE)
+      && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER) && DISABLED(PANEL_ONE) && DISABLED(MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER)
```

Comment:
This is not needed, unless enabling `MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER` were to define `ULTIMAKERCONTROLLER` which I don't see that it does.

Reply:
I got sanity check failure, two displays defined without it.

Reply:
#if defined (MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER)
 #define DOGLCD
 #define U8GLIB_SH1106
 #define REPRAP_DISCOUNT_SMART_CONTROLLER
 #define NEWPANEL
#endif
and 
#if ENABLED(PANEL_ONE) || ENABLED(U8GLIB_SH1106)
    #define ULTIMAKERCONTROLLER
  #endif
it is defined